kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -657,6 +657,20 @@ object KyuubiConf {
       .checkValue(_ > 0, "the maximum must be positive integer.")
       .createWithDefault(10)
 
+  val SESSION_ENGINE_SYNC_INIT: ConfigEntry[Boolean] =
+    buildConf("session.engine.sync.init")
+      .doc("When opening kyuubi session, whether to init backend engine synchronously")
+      .version("1.4.0")
+      .booleanConf.
+      createWithDefault(true)
+
+  val SESSION_ENGINE_STARTUP_CHECK_INTERVAL: ConfigEntry[Long] =
```

Comment:
The current engine startup check is hard coded, maybe we can just follow that just for now